### PR TITLE
Doc updates

### PR DESCRIPTION
**Pull request by foresmac**
In particular, noting that `s` should be an integer (see #123).

@rachelhyman @vokal/systems

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,8 +13,7 @@ Images are served up with a URI that contains a bucket name, as well as a unique
 
 When images are requested they are placed into an in-memory cache to make repeated requests for that image faster.
 
-You can resize an image on the fly by providing an `?s=X` parameter that specifies
-a maximum width for the image. Resized images may also be center-cropped by passing `?c=true` in the querystring. The resiezed image or thumbnail will then be cached to both `groupcache` and S3. If the image leaves the in-memory cache it will not need to be resized again.
+You can resize an image on the fly by providing an `?s=<integer>` parameter that specifies a maximum width for the image in pixels. Resized images may also be center-cropped by passing `?c=true` in the querystring. The resiezed image or thumbnail will then be cached to both `groupcache` and S3. If the image leaves the in-memory cache it will not need to be resized again.
```
Comment (adambain-vokal) [inline README.md]: SP resiezed

Comment (rachelhyman) [inline README.md]: Is the `<integer>` notation gonna confuse doofuses like me into passing in a literal parameter like `<270>`?

I might also put it in the text of this paragraph, like just calling it an "integer parameter" so it's clear.

**Comment by foresmac** [comment]
More better @rachelhyman?

**Comment by rachelhyman** [comment]
Me likey

**Comment by foresmac** [comment]
Tanks!

**Pull request merged**